### PR TITLE
Document how to update to a new batou 2.x version.

### LINKED ISSUE #103: Document how to update to a new batou release
Currently I edit the `requirements.txt` file and run `./batou appenv-update-lockfile` afterwards to update `requirements.lock`.

I am not sure if this is the canonical way and I did not find anything about that topic in the documentation at https://batou.readthedocs.io/en/stable/index.html
(Additionally the neither the documentation nor `./batou -h` mentions the `appenv-update-lockfile` subcommand.)

If my approach is the correct one, I could contribute a bit of documentation.

### PR DESCRIPTION
**Pull request by icemac**
Fixes #103.

### CODE REVIEW

File: doc/source/upgrading.txt [hunk 1]
```diff
@@ -68,3 +68,20 @@ Things you may have to change in your deployment
...
+
+Updating to a newer batou 2.x version
+=====================================
+
+After having switched from batou 1.x to 2.x you might update to the latest
+``batou`` release once in a while. This requires the following steps:
+
+* Edit ``requirements.txt`` and enter the desired version: ``batou==2.x``.
+
+    - If you want to use a not released version use a git URL like this:
```
Comment (ctheune) [inline doc/source/upgrading.txt:81]: wording: "not release" -> unreleased

**Review by ctheune — CHANGES_REQUESTED** [review]
Looks good! Minor wording issue that I'd like corrected, though.

File: doc/source/upgrading.txt (hunk 1 above)
Comment (icemac) [reply doc/source/upgrading.txt]: Fixed.

**Comment by icemac** [comment]
Hm, the tests are broken on GH Actions but it does not seem to be my fault.

**Comment by ctheune** [comment]
Yeah, no worries. I moved to GA earlier this week but didn't get everything done back then. I'll fix those later.

**Pull request merged**